### PR TITLE
docs: improve contributing.md and some minor improvements

### PR DESCRIPTION
**Pull request by chrislemke**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -51,3 +52,131 @@ The following steps will give a short guide on how to contribute to this project
...
+
+8. When you work on the implementation try to make the implementation as generic as possible. In other words, try to wrap a powerful Plotly implementation in your blitzly function and pass as many arguments as possible to the user while giving default arguments. In addition to this please also provide a `plotly_kwargs` argument which allows the user to pass additional keyword arguments to the Plotly function. This way the user can customize the plot as much as possible.
+
+9. Don't forget to write some meaningful unit tests. The tests should cover all possible use cases of the function and should be located in the `tests/test_cases` folder and should be named `test_<function_name_with_use_case>.py`.
+
+10. Documentation: Please add your new blitly ⚡️ plot to the list in the [`README.me`](https://github.com/invia-flights/blitzly#available-plots-so-far-) file. If you think your plot implementation is worth to be documented even more, feel free to add it together with a short description to the end of the [`playground.ipynb`](https://github.com/invia-flights/blitzly/blob/main/examples/playground.ipynb) notebook.
```
Comment (premsrii) [inline docs/CONTRIBUTING.md:180]:
```suggestion
10. Documentation: Please add your new blitly ⚡️ plot to the list in the [`README.md`]
```

Comment (premsrii) [inline docs/CONTRIBUTING.md:180]:
```suggestion
10. Documentation: Please add your new blitly ⚡️ plot to the list in the [`README.md`](https://github.com/invia-flights/blitzly#available-plots-so-far-) file. If you think your plot implementation is worth to be documented even more, feel free to add it together with a short description to the end of the [`playground.ipynb`](https://github.com/invia-flights/blitzly/blob/main/examples/playground.ipynb) notebook.
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -51,3 +52,131 @@ The following steps will give a short guide on how to contribute to this project
...
+
+That's it! 🎉 Happy ploding! 📊👩‍💻 If you have any questions feel free to contact us!
```
Comment (premsrii) [inline docs/CONTRIBUTING.md:182]:
```suggestion
That's it! 🎉 Happy plotting! 📊👩‍💻 If you have any questions feel free to contact us!
```

**Review by premsrii — CHANGES_REQUESTED** [review]
See other comments for minor typos.

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (chrislemke) [reply docs/CONTRIBUTING.md:182]: That was actually on purpose. It's supposed to be a mash-up of plotting and coding.

**Pull request merged**